src/components/Iwd/ActivityFeed.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+	<div class="tw-flex tw-gap-x-0.5 tw-overflow-x-auto tw-py-2 tw-px-1 hide-scrollbar">
+		<activity-card
+			v-for="(activity, index) in activities"
+			:key="index"
+			:activity="activity"
+		/>
+	</div>
+</template>
+
+<script>
+import ActivityCard from '@/components/Iwd/ActivityCard';
+import IwdActionsQuery from '@/graphql/query/IwdActions.graphql';
+
+export default {
+	name: 'ActivityFeed',
+	components: {
+		ActivityCard,
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			activities: [],
+		};
+	},
+	methods: {
+		fetchActivities() {
+			// Fetch activities from API
+			this.apollo.query({
+				query: IwdActionsQuery,
+			}).then(({ data }) => {
+				this.activities = data?.lend?.campaignActions?.values ?? [];
+			});
+		},
+	},
+	mounted() {
+		this.fetchActivities();
+	},
+};
+</script>
+
+<style scoped lang="postcss">
+.hide-scrollbar {
```

Comment:
Testing locally this means you can't really scroll on desktop. Once this is on dev, could you tag Nathan in our team channel about how he wants to handle this for desktop?

Reply:
Yeah, lets see what is happening with dynamic data and what limit we gonna consider for this...

Reply:
A good desktop UX is having arrows to navigate on it, but lets see what is happening then.